### DISCUSSION TITLE
[Question] Build a windows flet project with python version 3.9

### DISCUSSION THREAD

**Discussion by ali19991377** [flet-dev/flet#4572] [Q&A]
### Question

hi

i want to use python 3.9 instead of python 3.11 for building windows application.
Where i can specify the python version ? 

every sub version from 3.9 is ok for me.

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ali19991377** [comment]
is there any way or hack to run app on python version 3.9 ?
currently even when i set
 [tool.poetry.dependencies]
python = "^3.9"
for my project it doesn't run on python version 3.9.

is it even possible ?

**Reply by FeodorFitsner** [reply]
Nope, it's only Python 3.12 at the moment.
